x-pack/plugins/stack_alerts/server/rule_types/index_threshold/rule_type.ts

```diff
@@ -78,30 +77,93 @@ export function getRuleType(
     'xpack.stackAlerts.indexThreshold.actionVariableContextThresholdLabel',
     {
       defaultMessage:
-        "An array of values to use as the threshold; 'between' and 'notBetween' require two values, the others require one.",
+        'An array of rule threshold values. For between and notBetween thresholds, there are two values.',
```

Comment:
>        'An array of rule threshold values. For between and notBetween thresholds, there are two values.',

I didn't notice the "noBetween" option in the UI so if it's not actually supported we'll need to fix this string too.

Reply:
Resolved in this commit [1e38ffe](https://github.com/elastic/kibana/pull/161807/commits/1e38ffe201768463f45e8ee2359356db78f5dfbd)